utilities/_utilities.responsive-spacings.scss

```diff
@@ -0,0 +1,72 @@
+/* ==========================================================================
+   #RESPONSIVE-SPACINGS
+   ========================================================================== */
+
+/**
+ * Utility classes enhancing the normal spacing classes by adding responsiveness
+ * to them. By default this brings us the following classes:
+ *
+ *   .u-margin-bottom-none@mobile {}
+ *   .u-margin-bottom-none@tablet {}
+ *   .u-margin-bottom-none@desktop {}
+ *   .u-margin-bottom-none@wide {}
+ *
+ * But we can affect the classes that get generated by altering the appropriate
+ * Sass maps below.
+ */
+
+
+
+// When using Sass-MQ, this defines the separator for the breakpoints suffix
+// in the class name. By default, we are generating the responsive suffixes
+// for the classes with a `@` symbol so you get classes like:
+// <div class="u-margin-bottom@mobile">
+$inuit-widths-breakpoint-separator: \@ !default;
```

Comment:
Since we explain what the separator does by default, we should add why there is a `\` in place before the `@` symbol (-> Since the `@`-symbol is a reserved symbol in CSS, we need to escape it in order to work properly in classes).